clang/lib/CConv/ConstraintVariables.cpp

```diff
@@ -1174,8 +1151,8 @@ bool FunctionVariableConstraint::
       Ret = (numParams() == OtherFV->numParams());
       Ret = Ret && returnVars->solutionEqualTo(CS, OtherFV->returnVars);
       for (unsigned i=0; i < numParams(); i++) {
-        Ret = Ret && cvSetsSolutionEqualTo(CS, getParamVar(i),
```

Comment:
Do we still need the `cvSetSolutionEqualTo` function (didn't check, just wondering) ?